pkgs/top-level/perl-packages.nix

```diff
@@ -7245,7 +7245,7 @@ let self = _self // overrides; _self = with self; {
 
       license = with licenses; [ gpl1Plus /* or */ artistic2 ];
 
-      maintainers = [ ];
+      maintainers = [ kiloreux ];
```

Comment:
Your maintainer entry is not in scope here like this: https://travis-ci.org/NixOS/nixpkgs/jobs/264326247#L548

It is in `lib.maintainers`, so as we're in a `with stdenv.lib;` you could either use `[ maintainers.kiloreux ]` or `with maintainers; [ kiloreux ]`.